trailblazer/server/api.py

```diff
@@ -257,6 +257,28 @@ def put_set_analysis_uploaded():
         store.set_analysis_uploaded(
             case_id=content.get("case_id"), uploaded_at=content.get("uploaded_at")
         )
-        return jsonify("Success! Update request sent"), 201
+        return jsonify("Success! Uploaded at request sent"), 201
+    except Exception as error:
+        return jsonify(f"Exception: {error}"), 409
```

Comment:
If you want to it could be good too implement a general class to handle HTTP exceptions, see https://github.com/Clinical-Genomics/atlas/blob/master/atlas/io/http_exceptions.py